### PR TITLE
smaller app.go

### PR DESCRIPTION
**Pull request by faddat**
Closes: #671 

## Description

Today I worked on upgrading akash to v0.45.1 and got a feel for what @ValarDragon was describing for #671.  I thought that moving the blocked addresses and store keys out would aid the cause. 

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Review by ValarDragon — APPROVED** [review]
Thanks for this! Going to close the dependabot tendermint version bump PR!

Copying my comment from there:
This seems good to me looking at the TM changelog https://github.com/tendermint/tendermint/blob/v0.34.16/CHANGELOG.md#v0.34.16

But would appreciate someone more informed on the TM changes to opine. (In particular that https://github.com/tendermint/tendermint/pull/7885 isn't problematic with our app version being initially set wrong) Will leave open for a day or so, to give others a chance to take a look

The TM-DB change looks solid to me!

File: app/keys.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	gammtypes "github.com/osmosis-labs/osmosis/v7/x/gamm/types"
+	incentivestypes "github.com/osmosis-labs/osmosis/v7/x/incentives/types"
+	lockuptypes "github.com/osmosis-labs/osmosis/v7/x/lockup/types"
+	minttypes "github.com/osmosis-labs/osmosis/v7/x/mint/types"
+	poolincentivestypes "github.com/osmosis-labs/osmosis/v7/x/pool-incentives/types"
+	superfluidtypes "github.com/osmosis-labs/osmosis/v7/x/superfluid/types"
+	txfeestypes "github.com/osmosis-labs/osmosis/v7/x/txfees/types"
+)
+
+func KVStoreKeys() []string {
```
Comment (ValarDragon) [inline app/keys.go:30]: I think this belongs in keepers.go

**Review by ValarDragon — APPROVED** [review]
LGTM! I think that the store keys should go in keepers.go as well though

**Comment by faddat** [comment]
Ok I'll move the store keys to keepers.go and eliminate keys.go

**Pull request merged**